apps/utils/deletion.py

```diff
@@ -218,3 +222,27 @@ def get_related_pipelines_queryset(instance, pipeline_param_key: str = None):
         Q(**{f"params__{pipeline_param_key}": instance.id}) | Q(**{f"params__{pipeline_param_key}": str(instance.id)})
     )
     return pipelines
+
+
+def get_admin_emails(team):
+    from apps.teams.models import Membership
+
+    return list(
+        Membership.objects.filter(team__name=team.name, groups__permissions__codename="delete_team").values_list(
+            "user__email", flat=True
+        )
+    )
+
+
+def send_domain_deleted_notification(team_name, admin_emails):
```

Comment:
nit: `domain` -> `team`
```suggestion
def send_team_deleted_notification(team_name, admin_emails):
```